swift/extractor/translators/StmtTranslator.cpp

```diff
@@ -73,9 +73,11 @@ codeql::ForEachStmt StmtTranslator::translateForEachStmt(const swift::ForEachStm
   auto entry = dispatcher.createEntry(stmt);
   fillLabeledStmt(stmt, entry);
   entry.body = dispatcher.fetchLabel(stmt.getBody());
-  entry.sequence = dispatcher.fetchLabel(stmt.getTypeCheckedSequence());
+ // entry.sequence = dispatcher.fetchLabel(stmt.getTypeCheckedSequence());
```

Comment:
Hmm... Isn't the sequence the `xs` in:
```swift
for i in xs { /* ... */ }
```
do we no longer extract this expression then?

Reply:
Oh, looking at the PrintAST test I now see what's going on. Instead of being able to do `myLoop.getSequence()` you should now do `myLoop.getIteratorVar().getInit()`. Is that right?

Reply:
That actually makes the change note easier to write (and no need for a deprecation), so I'll go ahead and do it as part of this.